README.md

```diff
@@ -41,8 +41,7 @@ If you already have `kubectl` installed and you want to use it to access the mic
 microk8s.kubectl config view --raw > $HOME/.kube/config
 ```
 
-Note: microk8s uses the loopback address as the endpoint.  If you wish to access the cluster from another machine, you will need
-to edit the config to use the host machine's IP (or, if you're using edge, you can use the `microk8s.config` helper).
+Note: microk8s uses the loopback address as the endpoint.  If you wish to access the cluster from another machine, you will need to edit the config to use the host machine's IP (or, if you're using edge, you can use the `microk8s.config` helper).
```

Comment:
How would a person "edit the config to use the host machine's IP?" We should be specific here.

Also, isn't the microk8s.config command in beta now, not edge?